auth/webRoutes.py

```diff
@@ -114,5 +113,111 @@ def removeUser(userid):
     except HTTPRequestError as err:
         return formatResponse(err.errorCode, err.message)
 
+# Permission CRUD
+@app.route('/pap/crud/permission', methods=['POST'])
```

Comment:
I think it might be better to drop the "crud" portion of the url (it adds no semantic or operational meaning to the request, since we're following a restful approach)